src/context/category.context.jsx

```diff
@@ -0,0 +1,39 @@
+import { createContext, useContext, useEffect, useState } from "react";
+const axios = require("axios").default;
+
+
+
+const defaultValue = null;
+
+const contextOfCategories = createContext(defaultValue);
```

Comment:
Since this is a context I think you should name it ContextOfCategories instead of contextOfCategories